Ports/freeciv/package.sh

```diff
@@ -30,3 +30,11 @@ launcher_name='Freeciv'
 launcher_category='&Games'
 launcher_command='/usr/local/bin/freeciv-sdl2'
 icon_file='windows/client.ico'
+
+post_install() {
+    echo
+    echo "Freeciv version ${version} is installed!"
+    echo
+    echo "Release notes: https://freeciv.fandom.com/wiki/NEWS-${version}"
+    echo
+}
```

Comment:
Why though? Nobody else does this. :yakthonk:

Reply:
We do this for a few ports. I can drop it though if we do not want it.

Reply:
Or maybe it is just the DOSbox staging that has this exact output. (I added that as well). Let me know if you want me to drop it and I'll drop it.

Reply:
I'd rather remove it, "port X version Y is now installed" is already mentioned by the ports framework itself, and the release notes are just moderately helpful for someone who just installed the version in question (compared to someone who wants to know whether it's worth to make the effort and upgrade).

Reply:
Done